### PR TITLE
Blanket Search Integration

### PR DESCRIPTION
**Pull request by pyxld-kris**
Implements a new ML model that recognizes all Initializer NFT images, combining it with the demo drag and drop code from demo.blankhumanity.com

Access the new page via the route /BlanketSearch

### CODE REVIEW

File: src/components/common/BlanketDragZone/BlanketDragZone.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        pointerEvents: "none",
+        zIndex: 100000000,
+      }}
+    >
+      <input
+        style={{
+          width: "300vw",
```
Comment (petersgrandadventure) [inline src/components/common/BlanketDragZone/BlanketDragZone.js:24]: wouldn't anything above 100vw be unseen/unnecessary?

File: src/utils/blanket.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import search from "./search";
```
Comment (petersgrandadventure) [inline src/utils/blanket.js:1]: This file can be removed, yeah?

**Review by petersgrandadventure — APPROVED** [review]
Left a few small comments, feel free to address or leave as is and merge :)

File: src/components/common/BlanketDragZone/BlanketDragZone.js (hunk 1 above)
Comment (pyxld-kris) [reply src/components/common/BlanketDragZone/BlanketDragZone.js:24]: I did this a while back, but if I remember correctly it was just me being cautious to make sure it covered the entire screen with room to spare. I *think* there may have been an issue where one of the corners wasn't working with 100%, so I just cranked everything up to 300

File: src/utils/blanket/search/reverseImage/BlanketImage.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    return this.imageData;
+  }
+  toBase64() {
+    return this.toDataUri().replace(/^data:image\/(png|jpg|webp);base64,/, ""); // correct base64
+  }
+  toBuffer() {
+    return Buffer.from(this.toBase64(), "base64");
+  }
+
+  async toPerceptualHash(imageData) {
```
[the remark this answers is not in the document]
Comment (pyxld-kris) [reply src/utils/blanket/search/reverseImage/BlanketImage.js:78]: This whole file is unused currently, but I left this file in as a potential abstraction to implement in the future. I'll leave what you want to do with it in your hands

File: src/utils/blanket.js (hunk 2 above)
Comment (pyxld-kris) [reply src/utils/blanket.js:1]: Yeah, I think this was leftover from a refactor. Will remove it now

File: src/utils/blanket/search/reverseImage/BlanketImage.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (petersgrandadventure) [reply src/utils/blanket/search/reverseImage/BlanketImage.js:78]: ah ok, I believe you provided this to me previously and I pulled some of it into a utils file.

**Pull request merged**